configs/_base_/datasets/dota_coco.py

```diff
@@ -0,0 +1,96 @@
+# dataset settings
+dataset_type = 'mmdet.CocoDataset'
+data_root = 'data/split_ms_dota/'
+file_client_args = dict(backend='disk')
+
+train_pipeline = [
+    dict(type='mmdet.LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='mmdet.LoadAnnotations', with_bbox=True, with_mask=True),
```

Comment:
Can we set poly2mask=False to use polygon mask as default? It will be faster than bitmap mask.